Docs/user_guide/adaround.rst

```diff
@@ -1,84 +1,81 @@
 .. _ug-adaround:
 
 
-=====================
+##############
 AIMET AdaRound
-=====================
+##############
 
- AIMET quantization features, by default, use the "nearest rounding" technique for achieving quantization.
- In the following figure, a single weight value in a weight tensor is shown as an illustrative example. When using the
- "nearest rounding" technique, this weight value is quantized to the nearest integer value. The Adaptive Rounding
- (AdaRound) feature, uses a smaller subset of the unlabelled training data to adaptively round the weights of modules
- with weights. In the following figure, the weight value is quantized to the integer value far from it. AdaRound,
- optimizes a loss function using the unlabelled training data to adaptively decide whether to quantize a specific
- weight to the integer value near it or away from it. Using the AdaRound quantization, a model is able to achieve an
- accuracy closer to the FP32 model, while using low bit-width integer quantization.
-
- When creating a QuantizationSimModel using the AdaRounded model, use the QuantizationSimModel provided API for
- setting and freezing parameter encodings before computing the encodings. Please refer the code example in the AdaRound
- API section.
+By default, AIMET uses *nearest rounding* for quantization. A single weight value in a weight tensor is illustrated in the following figure. In nearest rounding, this weight value is quantized to the nearest integer value.
 
+The Adaptive Rounding (AdaRound) feature uses a subset of the unlabeled training data to adaptively round weights. In the following figure, the weight value is quantized to the integer value far from it.
 
 .. image:: ../images/adaround.png
     :width: 900px
 
-AdaRound Use Cases
-=====================
+AdaRound optimizes a loss function using the unlabelled training data to decide whether to quantize a weight to the closer or further integer value. AdaRound quantization acieves accuracy closer to the FP32 model using low bit-width integer quantization.
```

Comment:
Typo: `Achieves`

Reply:
Suggestion:

Change: "AdaRound quantization achieves accuracy closer to the FP32 model using low bit-width integer quantization."

To: "AdaRound quantization achieves accuracy closer to the FP32 model, while using low bit-width integer quantization"

Reply:
Changed.